schema/home/fetch.js

```diff
@@ -4,7 +4,7 @@ import { sortBy, first, forEach, clone } from 'lodash';
 import blacklist from '../../lib/artist_blacklist';
 
 export const featuredFair = () => {
-  return gravity('fairs', { size: 5, status: 'running', has_full_feature: true }).then((fairs) => {
+  return gravity('fairs', { size: 5, active: true, has_full_feature: true }).then((fairs) => {
```

Comment:
currently active fairs that are not published but have a full feature with no microsite but a year round page and no organizer profile but a fair profile....and go!